src/twisted/test/test_defer.py

```diff
@@ -387,15 +387,15 @@ def testDeferredListDontConsumeErrors(self) -> None:
         errorTrap: List[Failure] = []
         d1.addErrback(errorTrap.append)
 
-        resultLists: List[_DeferredListResultListT] = []
+        resultLists: List[_DeferredListResultListT[None]] = []
         dl.addCallback(resultLists.append)
 
         d1.errback(GenericError("Bang"))
         self.assertEqual("Bang", errorTrap[0].value.args[0])
         self.assertEqual(1, len(resultLists))
         firstResult = resultLists[0][0]
         assert firstResult is not None
-        self.assertEqual("Bang", firstResult[1].value.args[0])
+        self.assertEqual("Bang", firstResult[1].value.args[0])  # type: ignore[attr-defined]
```

Comment:
this show the type annotation for DeferredList is working now, but the type annotation says the result can never be a Failure.

This lie works for Deferred but cause attr-defined errors for DeferredList

Reply:
thanks. we can keep the ignore for now.